docs/contributing.md

```diff
@@ -0,0 +1,11 @@
+# Code Review
+
+The following checklist should be used during code reviews:
+
+- [ ] Code accomplishes its functional purpose.
+- [ ] Code passes automated testing (none at the moment)
+- [ ] Code passes linting (in progress).
```

Comment:
Just to make sure but does linting work to standardize the styling of our code. If not, we should add to the check list something to do about style guides. @rahuselama @yum-feng

Reply:
yeah, thats correct, @adrianadanko do you thin you are ready to integrate linting into the project ?

Reply:
linting does a very good job at standardizing coding styles, and can be configured according to the developers taste.

I was able to run a checkstyle lint, here's a demonstration of a lint output:

```text
[ant:checkstyle] [WARN] /home/augustfengd/repositories/gh/yum-feng/COMP5541TeamZombieCode/src/main/java/teamzombie/pokez/PokeZ.java:4:1: Wrong lexicographical order for 'net.minecraftforge.fml.common.Mod' import. Should be before 'teamzombie.pokez.setup.Registration'. [CustomImportOrder]
[ant:checkstyle] [WARN] /home/augustfengd/repositories/gh/yum-feng/COMP5541TeamZombieCode/src/main/java/teamzombie/pokez/PokeZ.java:9:1: Line contains a tab character. [FileTabCharacter]
[ant:checkstyle] [WARN] /home/augustfengd/repositories/gh/yum-feng/COMP5541TeamZombieCode/src/main/java/teamzombie/pokez/PokeZ.java:10:5: 'member def modifier' has incorrect indentation level 4, expected level should be 2. [Indentation]
[ant:checkstyle] [WARN] /home/augustfengd/repositories/gh/yum-feng/COMP5541TeamZombieCode/src/main/java/teamzombie/pokez/PokeZ.java:12:5: 'ctor def modifier' has incorrect indentation level 4, expected level should be 2. [Indentation]
[ant:checkstyle] [WARN] /home/augustfengd/repositories/gh/yum-feng/COMP5541TeamZombieCode/src/main/java/teamzombie/pokez/PokeZ.java:12:5: Missing a Javadoc comment. [MissingJavadocMethod]
[ant:checkstyle] [WARN] /home/augustfengd/repositories/gh/yum-feng/COMP5541TeamZombieCode/src/main/java/teamzombie/pokez/PokeZ.java:13:5: Line contains a tab character. [FileTabCharacter]
[ant:checkstyle] [WARN] /home/augustfengd/repositories/gh/yum-feng/COMP5541TeamZombieCode/src/main/java/teamzombie/pokez/PokeZ.java:14:5: Line contains a tab character. [FileTabCharacter]
[ant:checkstyle] [WARN] /home/augustfengd/repositories/gh/yum-feng/COMP5541TeamZombieCode/src/main/java/teamzombie/pokez/PokeZ.java:15:5: Line contains a tab character. [FileTabCharacter]
[ant:checkstyle] [WARN] /home/augustfengd/repositories/gh/yum-feng/COMP5541TeamZombieCode/src/main/java/teamzombie/pokez/PokeZ.java:15:9: 'ctor def' child has incorrect indentation level 8, expected level should be 4. [Indentation]
[ant:checkstyle] [WARN] /home/augustfengd/repositories/gh/yum-feng/COMP5541TeamZombieCode/src/main/java/teamzombie/pokez/PokeZ.java:16:1: Line contains a tab character. [FileTabCharacter]
[ant:checkstyle] [WARN] /home/augustfengd/repositories/gh/yum-feng/COMP5541TeamZombieCode/src/main/java/teamzombie/pokez/PokeZ.java:17:5: 'ctor def rcurly' has incorrect indentation level 4, expected level should be 2. [Indentation
```

@adrianadanko I did a bit of research on linting and have gotten something going, https://github.com/yum-feng/COMP5541TeamZombieCode/pull/15. I think the next steps are to find a way run the tasks via command line and then we can integrate it into the CI/CD.